### PR TITLE
PIP-206 Reorganise pom to help centralise dependency management in th…

### PR DESCRIPTION
**Pull request by jimcornmell**
Changes to pom in preparation to make BOM file for the future CVE work.  If this is approved I'll use it as a starting point to migrate Spring boot to 2.7.8.

### CODE REVIEW

**Review by CoderMChu — APPROVED** [review]
@jimcornmell It looks good. Thanks for tidying up these.

File: api/pom.xml [hunk 1]
```diff
@@ -49,7 +44,6 @@
 		<dependency>
 			<groupId>org.springframework.boot</groupId>
 			<artifactId>spring-boot-starter-test</artifactId>
-			<scope>test</scope>
```
Comment (pgwilliams) [inline api/pom.xml:52]: Jim, is the test scope definitely inherited from the parent pom as well as the version, or does it need to be repeated each time?   Thanks!

Comment (jimcornmell) [reply api/pom.xml:52]: It is (just tried) but also it can be overridden locally if needed (tried that too).

Comment (pgwilliams) [reply api/pom.xml:52]: 👍  thanks for checking.

**Pull request merged**